src/test/java/org/sagebionetworks/web/unitclient/widget/entity/MarkdownEditorWidgetTest.java

```diff
@@ -271,24 +271,34 @@ public void testResizeMarkdown() {
     String markdown = "";
 
     when(mockView.getClientHeight()).thenReturn(0);
+    when(mockView.getClientWidth()).thenReturn(1000);
     presenter.resizeMarkdownTextArea();
     verify(mockView)
       .setMarkdownTextAreaHeight(MarkdownEditorWidget.MIN_TEXTAREA_HEIGHT);
 
     reset(mockView);
+    when(mockView.getClientWidth()).thenReturn(1000);
     when(mockView.getClientHeight())
       .thenReturn(MarkdownEditorWidget.MIN_TEXTAREA_HEIGHT);
     presenter.resizeMarkdownTextArea();
     verify(mockView)
       .setMarkdownTextAreaHeight(MarkdownEditorWidget.MIN_TEXTAREA_HEIGHT);
 
     reset(mockView);
+    when(mockView.getClientWidth()).thenReturn(1000);
     when(mockView.getClientHeight()).thenReturn(1000);
     presenter.resizeMarkdownTextArea();
     verify(mockView)
       .setMarkdownTextAreaHeight(
         1000 - MarkdownEditorWidget.OTHER_EDITOR_COMPONENTS_HEIGHT
       );
+
+    // For mobile screen sizes (below XS_BREAKPOINT), set height to 50% of client height
+    reset(mockView);
+    when(mockView.getClientWidth()).thenReturn(400);
```

Comment:
I would reference the breakpoint const value.  Something like (XS_BREAKPOINT-100)